tests/wlr_layer_shell_v1.cpp

```diff
@@ -41,6 +41,11 @@ class LayerSurfaceTest: public wlcs::StartedInProcessServer
     {
     }
 
+    ~LayerSurfaceTest()
+    {
+        client.roundtrip();
```

Comment:
I don't see how this can have any effect?

The client only ever gets callbacks from the server when explicitly dispatched; this adds an explicit dispatch point, but without this explicit dispatch all that will be happening in `~LayerSurfaceTest` is ~LayerSurfaceV1()`, `~Surface()`, then `~Client()`, none of which perform dispatch?